### DISCUSSION TITLE
{{.Arch}} only available when running nfpm in goreleaser?

### DISCUSSION THREAD

**Discussion by qrkourier** [goreleaser/nfpm#697] [General]
I'm grateful for this project. It's saved me loads of time. Thank you!

It's mentioned in the commented config sample for standalone nfpm that templating won't be supported. Is that why `{{.Arch}}` is available in nfpm only when the nfpm config is nested in a goreleaser config?

I was hoping to use `{{.Arch}}` in the the standalone nfpm and it seemed intuitive that it would be available in the `contents` context the way it's demonstrated in the nested goreleaser config sample here: https://goreleaser.com/customization/nfpm/. I wasn't able to use the env var interpolation feature of standalone nfpm either because it's not available in the `contents` context. I worked around this by moving the contents src file to a predictable location before running nfpm.

**Comment by caarlos0** [comment]
Hey, the documentation for nfpm, as a binary, can be found at https://nfpm.goreleaser.com

The nFPM integration in GoReleaser both adds and hides things from the actual nfpm config file.

Hope that helps
